sdk/js/src/Client.ts

```diff
@@ -723,14 +722,13 @@ export class DevCycleClient<
                     true,
                 )
             ) {
-                saveEntity(
+                const res = await saveEntity(
                     this.user,
                     this.sdkKey,
                     this.logger,
                     this.options,
-                ).then((res) =>
-                    this.logger.info(`Saved response entity! ${res}`),
                 )
+                this.logger.info(`Saved response entity! ${res}`)
             }
 
             this.userSaved = true
```

Comment:
what are the implications of "this.userSaved" not being synchronously set to true as soon as `setUser` is called, like it was before?

Reply:
great question, ill look into that

Reply:
Looks like its only used to prevent this function from being called multiple times unnecessarily. Dont think theres any real issue here